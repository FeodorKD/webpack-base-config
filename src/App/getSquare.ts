export const getSquare = (number: number): number => {
    return number * number;
};
