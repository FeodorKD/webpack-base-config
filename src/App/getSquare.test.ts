import { getSquare } from './getSquare';

describe('Тестирование функции получения квадрата числа', () => {
    test('getSquare', () => {
        expect(getSquare(5)).toBe(25);
        expect(25).toMatchSnapshot();
    });
    test('getSquare', () => {
        expect(getSquare(-5)).toBe(25);
        expect(25).toMatchSnapshot();
    });
    test('getSquare', () => {
        expect(getSquare(0)).toBe(0);
        expect(0).toMatchSnapshot();
    });
});
