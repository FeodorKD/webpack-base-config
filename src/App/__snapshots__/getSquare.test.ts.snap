// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Тестирование функции получения квадрата числа getSquare 1`] = `25`;

exports[`Тестирование функции получения квадрата числа getSquare 2`] = `25`;

exports[`Тестирование функции получения квадрата числа getSquare 3`] = `0`;
